src/internal.c

```diff
@@ -6908,28 +6922,14 @@ int InitSSL(WOLFSSL* ssl, WOLFSSL_CTX* ctx, int writeDup)
         XMEMSET(ssl->param, 0, sizeof(WOLFSSL_X509_VERIFY_PARAM));
 #endif
 
-#ifdef SINGLE_THREADED
```

Comment:
Do we still have an build option to support SINGLE_THREADED use for memory constrained devices? I don't see it.

Reply:
I don't think there is any reason to restrict this optimization just to resource constrained devices. We always try to reuse the context's `Suites` instance. `SINGLE_THREADED` still does away with the mutex locking logic for example so it is still useful for small devices.